<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDlg</class>
 <widget class="QDialog" name="ColorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>130</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourse.qrc">
    <normaloff>:/ico/icon/logo.png</normaloff>:/ico/icon/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>114</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RGB:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRgb"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbRGB">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourse.qrc">
         <normaloff>:/ico/icon/copy.png</normaloff>:/ico/icon/copy.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>HSV:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDEC"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbDEC">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourse.qrc">
         <normaloff>:/ico/icon/copy.png</normaloff>:/ico/icon/copy.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>HEX:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditHex"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbHex">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourse.qrc">
         <normaloff>:/ico/icon/copy.png</normaloff>:/ico/icon/copy.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tbPickOther">
       <property name="text">
        <string>拾取其他颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tbClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
